Makefile

```diff
@@ -1,3 +1,5 @@
+IMAGE_REPOSITORY?=app-sre
```

Comment:
We've been doing [this](https://github.com/openshift/boilerplate/blob/d9d5abfd0745651959b5e4de82b751ec4c3a3dac/boilerplate/openshift/golang-osd-operator/project.mk#L6) forever. Why did it stop working?